docs/backends/risingwave.qmd

```diff
@@ -0,0 +1,103 @@
+# RisingWave
+
+[https://risingwave.com/](https://risingwave.com/)
+
+![](https://img.shields.io/badge/memtables-fallback-yellow?style=flat-square) ![](https://img.shields.io/badge/inputs-Postgres tables-blue?style=flat-square) ![](https://img.shields.io/badge/outputs-Postgres tables | CSV | pandas | Parquet | PyArrow-orange?style=flat-square)
+
+## Install
+
+Install Ibis and dependencies for the RisingWave backend:
+
+::: {.panel-tabset}
+
+## `pip`
+
+Install with the `risingwave` extra:
+
+```{.bash}
+pip install 'ibis-framework[risingwave]'
+```
+
+And connect:
+
+```{.python}
+import ibis
+
+con = ibis.risingwave.connect()  # <1>
+```
+
+1. Adjust connection parameters as needed.
+
+## `conda`
+
+Install for Risingwave:
+
+```{.bash}
+conda install -c conda-forge ibis-risingwave
+```
+
+And connect:
+
+```{.python}
+import ibis
+
+con = ibis.risingwave.connect()  # <1>
+```
+
+1. Adjust connection parameters as needed.
+
+## `mamba`
+
+Install for Risingwave:
+
+```{.bash}
+mamba install -c conda-forge ibis-risingwave
+```
+
+And connect:
+
+```{.python}
+import ibis
+
+con = ibis.risingwave.connect()  # <1>
+```
+
+1. Adjust connection parameters as needed.
+
+:::
+
+## Connect
+
+### `ibis.risingwave.connect`
+
+```python
+con = ibis.risingwave.connect(
+    user="username",
+    password="password",
+    host="hostname",
+    port=4566,
+    database="database",
+)
+```
+
+::: {.callout-note}
+`ibis.risingwave.connect` is a thin wrapper around [`ibis.backends.risingwave.Backend.do_connect`](#ibis.backends.risingwave.Backend.do_connect).
+:::
+
+
+### Connection Parameters
```

Comment:
this should be "Connection parameters" per style guide (but seeing it's this for all the other backends, so maybe keep + fix this in a future PR)

Reply:
yeah, can handle that in a followup